js/gemini.js

```diff
@@ -712,6 +715,57 @@ module.exports = class gemini extends Exchange {
         return this.safeBalance (result);
     }
 
+    async fetchTradingFees (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetV1Notionalvolume (params);
```

Comment:
Should be  `await this.privatePostV1Notionalvolume (params);` instead of `await this.privateGetV1Notionalvolume (params);`

Currently crashing because a Get version does not exist